lib/acts_as_list/active_record/acts/list.rb

```diff
@@ -82,27 +82,27 @@ def insert_at!(position = acts_as_list_top)
 
         # Swap positions with the next lower item, if one exists.
         def move_lower
-          return unless lower_item
+          return unless (other = lower_item)
```

Comment:
Despite 15 odd years experience with Ruby, this one still made me double-take :) It's succinct for sure, but subtle enough that I initially read it as a comparison. I know it's less elegant but I think something like:

```
other_item = lower_item
return unless other_item
```

reads clearer. The other option would be wrapping the whole thing in an `if`.

Is there any kind of style guide for something like this?

My other question is whether ActiveRecord already caches this query? I've not checked, but it should show that in the logs, so while there's a lot of log noise, it's all just hits to the cached result? :)